contentcuration/contentcuration/views.py

```diff
@@ -14,7 +14,7 @@
 from kolibri.content.models import File
 
 def base(request):
-    return redirect(channel_list)    # redirect to the channel list page
+    return redirect('channels')    # redirect to the channel list page
```

Comment:
Oh, interesting -- didn't know it will [automatically apply reverse](https://docs.djangoproject.com/en/1.9/topics/http/shortcuts/#redirect). Seems like it could lead to ambiguous cases (since a relative URL could look like a url pattern name).

Good to go, then!